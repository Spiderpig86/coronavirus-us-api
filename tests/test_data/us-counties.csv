date,county,state,fips,cases,deaths
2020-01-21,Snohomish,Washington,53061,1,0
2020-01-22,Snohomish,Washington,53061,1,0
2020-01-23,Snohomish,Washington,53061,1,0
2020-01-24,Cook,Illinois,17031,1,0
2020-01-24,Snohomish,Washington,53061,1,0
2020-01-25,Orange,California,06059,1,0
2020-01-25,Cook,Illinois,17031,1,0
2020-01-25,Snohomish,Washington,53061,1,0
2020-01-26,Maricopa,Arizona,04013,1,0
2020-01-26,Los Angeles,California,06037,1,0
2020-01-26,Orange,California,06059,1,0
2020-01-26,Cook,Illinois,17031,1,0
2020-01-26,Snohomish,Washington,53061,1,0
2020-01-27,Maricopa,Arizona,04013,1,0
2020-01-27,Los Angeles,California,06037,1,0
2020-01-27,Orange,California,06059,1,0
2020-01-27,Cook,Illinois,17031,1,0
2020-01-27,Snohomish,Washington,53061,1,0
2020-01-28,Maricopa,Arizona,04013,1,0
2020-01-28,Los Angeles,California,06037,1,0
2020-01-28,Orange,California,06059,1,0
2020-01-28,Cook,Illinois,17031,1,0
2020-01-28,Snohomish,Washington,53061,1,0
2020-01-29,Maricopa,Arizona,04013,1,0
2020-01-29,Los Angeles,California,06037,1,0
2020-01-29,Orange,California,06059,1,0
2020-01-29,Cook,Illinois,17031,1,0
2020-01-29,Snohomish,Washington,53061,1,0
2020-01-30,Maricopa,Arizona,04013,1,0
2020-01-30,Los Angeles,California,06037,1,0
2020-01-30,Orange,California,06059,1,0
2020-01-30,Cook,Illinois,17031,2,0
2020-01-30,Snohomish,Washington,53061,1,0
2020-01-31,Maricopa,Arizona,04013,1,0
2020-01-31,Los Angeles,California,06037,1,0
2020-01-31,Orange,California,06059,1,0
2020-01-31,Santa Clara,California,06085,1,0
2020-01-31,Cook,Illinois,17031,2,0
2020-01-31,Snohomish,Washington,53061,1,0
2020-02-01,Maricopa,Arizona,04013,1,0
2020-02-01,Los Angeles,California,06037,1,0
2020-02-01,Orange,California,06059,1,0
2020-02-01,Santa Clara,California,06085,1,0
2020-02-01,Cook,Illinois,17031,2,0
2020-02-01,Suffolk,Massachusetts,25025,1,0
2020-02-01,Snohomish,Washington,53061,1,0