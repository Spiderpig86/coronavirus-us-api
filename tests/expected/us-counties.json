  [
    {
      "id": "US@Washington@Snohomish@53061",
      "country": "US",
      "state": "Washington",
      "county": "Snohomish",
      "fips": "53061",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-21T00:00:00Z": 1,
            "2020-01-22T00:00:00Z": 1,
            "2020-01-23T00:00:00Z": 1,
            "2020-01-24T00:00:00Z": 1,
            "2020-01-25T00:00:00Z": 1,
            "2020-01-26T00:00:00Z": 1
          },
          "latest": 1
        },
        "deaths": {
          "history": {
            "2020-01-21T00:00:00Z": 0,
            "2020-01-22T00:00:00Z": 0,
            "2020-01-23T00:00:00Z": 0,
            "2020-01-24T00:00:00Z": 0,
            "2020-01-25T00:00:00Z": 0,
            "2020-01-26T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 1,
        "deaths": 0
      }
    },
    {
      "id": "US@Illinois@Cook@17031",
      "country": "US",
      "state": "Illinois",
      "county": "Cook",
      "fips": "17031",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-24T00:00:00Z": 1,
            "2020-01-25T00:00:00Z": 1,
            "2020-01-26T00:00:00Z": 1
          },
          "latest": 1
        },
        "deaths": {
          "history": {
            "2020-01-24T00:00:00Z": 0,
            "2020-01-25T00:00:00Z": 0,
            "2020-01-26T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 1,
        "deaths": 0
      }
    },
    {
      "id": "US@California@Orange@06059",
      "country": "US",
      "state": "California",
      "county": "Orange",
      "fips": "06059",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-25T00:00:00Z": 1,
            "2020-01-26T00:00:00Z": 1
          },
          "latest": 1
        },
        "deaths": {
          "history": {
            "2020-01-25T00:00:00Z": 0,
            "2020-01-26T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 1,
        "deaths": 0
      }
    },
    {
      "id": "US@Arizona@Maricopa@04013",
      "country": "US",
      "state": "Arizona",
      "county": "Maricopa",
      "fips": "04013",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-26T00:00:00Z": 1
          },
          "latest": 1
        },
        "deaths": {
          "history": {
            "2020-01-26T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 1,
        "deaths": 0
      }
    },
    {
      "id": "US@California@Los Angeles@06037",
      "country": "US",
      "state": "California",
      "county": "Los Angeles",
      "fips": "06037",
      "timelines": {
        "confirmed": {
          "history": {
            "2020-01-26T00:00:00Z": 1
          },
          "latest": 1
        },
        "deaths": {
          "history": {
            "2020-01-26T00:00:00Z": 0
          },
          "latest": 0
        }
      },
      "last_updated": "2020-05-30T09:19:06Z",
      "latest": {
        "confirmed": 1,
        "deaths": 0
      }
    }
  ]